### ISSUE TITLE
WebUI if is notresponsive on mobile

### ISSUE THREAD

**Issue by cdn4lf** [nicolargo/glances#3059]
**Describe the bug**
In web server move (-w), the interface is not being mobile responsive and is showing as a jumbled mess.

**To Reproduce**
1. Start Glances with the following options '-w' using docker (4.3.0.5)
2. Open on mobile
3. See mess of information

**Expected behavior**
Should default to single column

![Screenshot_20241229_095740_Chrome](https://github.com/user-attachments/assets/6d408c8e-51c7-483d-b6ab-078500dc3be7)

**Comment by arpitgupta** [comment]
I have seen the same behavior. If i downgrade to 4.2 then it is better. Current UI works better if I put my phone in landscape mode. Still have to scroll items to be able to view drive temps etc. 

I am on android and tried this with chrome, firefox and brave browsers and they all show the same behavior.

**Comment by benjimons** [comment]
Same problem on iOS using Safari and Chrome for iOS.

From install of glances--4.3.0.5.arm64_sequoia.bottle.tar.gz

**Comment by nicolargo** [comment]
Hi all,

the WebUI has been completely refactor in this version. For the moment it is not responsive on Mobile (aka non landscape) screen.

A ugly work-around is to switch to "Computer version" on your Web browser. 

Any contributor are welcome to make the UX design fully responsive.

**Comment by nicolargo** [comment]
Have a look on:
* https://getbootstrap.com/docs/4.0/utilities/display/#hiding-elements
* https://getbootstrap.com/docs/5.0/layout/breakpoints/#available-breakpoints

**Comment by nicolargo** [comment]
@cdn4lf @arpitgupta @benjimons Please can you test the latest version (4.3.0.7), it should correct the issue.

If it does, can you send me back a screenshot ?

**Comment by cdn4lf** [comment]
@nicolargo , looks good to me.

![Screenshot_20250103_064426_Chrome](https://github.com/user-attachments/assets/0db46f8e-c362-44b6-b6d7-3367fccf2307)

**Comment by arpitgupta** [comment]
@nicolargo  it's better but now we seem to be missing data.

On left left frame data such as networks/disk io, filesystems and sensors is not visible at all.

Even on the top header we seem to be missing detailed data of memory, swap, cpu and load etc.

![Screenshot_20250103-064754](https://github.com/user-attachments/assets/e81e2a74-fee6-4dc7-b43e-0a6bbd2bb31c)

**Comment by nicolargo** [comment]
> @nicolargo  it's better but now we seem to be missing data.
> 
> On left left frame data such as networks/disk io, filesystems and sensors is not visible at all. I tried to see if I could scroll to find it but no luck.
> 
> Even on the top header we seem to be missing detailed data of memory, swap, cpu and load etc.
> 
> ![Screenshot_20250103-064754](https://github.com/user-attachments/assets/e81e2a74-fee6-4dc7-b43e-0a6bbd2bb31c)
> 

Yes it's not a bug. If you want to display all stats please use the Display as computer version of your Web browser.

**Comment by arpitgupta** [comment]
@nicolargo I do not see the the left frame  show desktop version is enabled in firefox. Top header data is visible.

**Comment by nicolargo** [comment]
I just push a new version on the develop branch, can you have a look ?

**Comment by arpitgupta** [comment]
Tried it out and the desktop view has more info now. See the various information from the left frame.

Some data is missing from the top header.  For example CPU info is shown but not all. Same for swap info etc.


This is the top header when viewing as a desktop site on mobile
![top-header](https://github.com/user-attachments/assets/69dd295e-8323-41c4-96e6-fad41004a89a)


vs what I see on actual desktop
![top-header-on-desktop](https://github.com/user-attachments/assets/44cb321b-7836-4c15-8602-395709784b07)

**Comment by nicolargo** [comment]
Yes space is limited. Only importants stats are displayed.
